cinn/utils/random_engine.cc

```diff
@@ -18,21 +18,21 @@ namespace cinn {
 namespace utils {
 
 // Sample Integers from uniform distribution [min, max)
-int SampleUniformInt(int min, int max, LinearRandomEngine::StateType* state) {
+int SampleUniformInt(int min, int max, LinearRandomEngine::StateType* rand_seed) {
   CHECK(min < max) << "Input value error: min(" << min << ") must be less than max(" << max << ")";
   if (min + 1 == max) {
     return min;
   }
 
-  LinearRandomEngine engine(state);
+  LinearRandomEngine engine(rand_seed);
   std::uniform_int_distribution<> dist(min, max - 1);
```

Comment:
```suggestion
  std::uniform_int_distribution<> dist(min, max);
```
为何不直接用闭区间？uniform_int_distribution支持min<=max